spring-integration-feed/src/main/java/org/springframework/integration/feed/inbound/FeedEntryMessageSource.java

```diff
@@ -178,6 +179,24 @@ private SyndEntry getNextEntry() {
 	private void populateEntryList() {
 		SyndFeed syndFeed = this.getFeed();
 		if (syndFeed != null) {
+			
+			/* 
+			 * 
+			 * QUESTION:
+			 * shouldn't there be a check here to see if the feed itself has been 
+			 * published/updated since last check? rfc4287 states that "atom:feed 
+			 * elements MUST contain exactly one atom:updated element." SyndFeed
+			 * doesnt seem to have a #getUpdatedDate method but since published
+			 * date is optional in spec, maybe the #getPublishDate takes care of 
+			 * this? My guess that why this is not happening is due to some oddity
+			 * with different implementations of the ATOM specification?
+			 * 
+			 * if (syndFeed.getPublistDate().getTime() > this.lastTime) {
+			 *   ...
+			 * }
```

Comment:
The code isn't a place for questions. 
Right `TODO` might be good choice, but according to your conclusion, we don't do that because not a good implementation for the `getPublishDate`. So, it's better to process `entries` and take a look to their date to compare with `this.lastTime`.

Any objection to leave this comment here?